src/registrar/forms/domain.py

```diff
@@ -213,6 +213,9 @@ class AuthorizingOfficialContactForm(ContactForm):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
+        # Overriding bc phone not required in this form
+        self.fields["phone"] = forms.IntegerField(required=False)
+
```

Comment:
(Q) Did you try changing this on the underlying ContactForm object?
It seems like doing that may be more complicated so I like this approach, though I wonder if that breaks anything or not

Reply:
It seems like in `domain.py`, only the Authorizing Official inherits `ContactForm`, however I was hesitant on changing it there in case in the future we'll have another form inherit that function (and also just in case it breaks something we're not aware of) 😅 Great call out though!